docs/book/src/topics/vpc/creating-a-cluster.md

```diff
@@ -1 +1,163 @@
 # Creating a cluster
+
+### Provision local boostrap management cluster:
+
+1. Create simple, local bootstrap cluster with a control-plane and worker node
+
+    Using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/):
+
+    ```console
+    ~ kind create cluster --name my-bootstrap --config bootstrap.yaml
+    ```
+
+    Example bootstrap.yaml:
+    ```yaml
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    nodes:
+       - role: control-plane
+       - role: worker
+    ```
+
+    Make sure the nodes are in `Ready` state before moving on.
+
+    ```console
+    ~ kubectl get nodes
+    NAME                         STATUS   ROLES                  AGE   VERSION
+    my-bootstrap-control-plane   Ready    control-plane,master   46h   v1.20.2
+    my-bootstrap-worker          Ready    <none>                 46h   v1.20.2
+    ```
+
+2. Apply IBM VPC CAPI CRDs
+
+    ```console
+    ~ kubectl apply -f config/crd/bases
+    ```
+
+    Output:
+    ```console
+    customresourcedefinition.apiextensions.k8s.io/ibmvpcclusters.infrastructure.cluster.x-k8s.io created
+    customresourcedefinition.apiextensions.k8s.io/ibmvpcmachines.infrastructure.cluster.x-k8s.io created
+    customresourcedefinition.apiextensions.k8s.io/ibmvpcmachinetemplates.infrastructure.cluster.x-k8s.io created
+    ```
+
+3. Initialize local bootstrap cluter as a management cluster
+
+    This cluster will be used to provision a workload cluster in IBM Cloud.
+
+    ```console
+    ~ clusterctl init
+    ```
+
+    Output:
+    ```console
+    Fetching providers
+    Installing cert-manager Version="v1.1.0"
+    Waiting for cert-manager to be available...
+    Installing Provider="cluster-api" Version="v0.3.16" TargetNamespace="capi-system"
+    Installing Provider="bootstrap-kubeadm" Version="v0.3.16" TargetNamespace="capi-kubeadm-bootstrap-system"
+    Installing Provider="control-plane-kubeadm" Version="v0.3.16" TargetNamespace="capi-kubeadm-control-plane-system"
+
+    Your management cluster has been initialized successfully!
+
+    You can now create your first workload cluster by running the following:
+
+      clusterctl config cluster [name] --kubernetes-version [version] | kubectl apply -f -
+    ```
+
+### Provision Workload Cluster in IBM Cloud VPC
+
+1. Set workload cluster environment variables
+
+    The sample IAM_ENDPOINT below points to Production and the SERVICE_ENDPOINT points to the `us-east` VPC region. Make sure these values reflect your target VPC environment in IBM Cloud.
+
+    ```console
+    export IAM_ENDPOINT=https://iam.cloud.ibm.com/identity/token
+    export SERVICE_ENDPOINT=https://us-south.iaas.cloud.ibm.com/v1
+    export API_KEY=<YOUR_API_KEY>
+    ```
+
+2. Run IBM provider controllers
+
+    The controllers will run against your local management bootstrap cluster.
+
+    ```console
+    ~ make run
+    ```
+
```

Comment:
These instructions aren't required when you have already deployed the controller part of `bootstrap` section